MediaBrowser.Providers/MediaInfo/ProbeProvider.cs

```diff
@@ -141,18 +141,18 @@ public bool HasChanged(BaseItem item, IDirectoryService directoryService)
                 && item.SupportsLocalMetadata
                 && !video.IsPlaceHolder)
             {
-                if (!video.SubtitleFiles.SequenceEqual(
+                if (!video.SubtitleFiles.Order().SequenceEqual(
```

Comment:
Rather than ordering and calling distinct, it might be more efficient to use Set operations

```C#
                var set = new HashSet(video.SubtitleFiles);
                set.SymmetricExceptWith( _subtitleResolver.GetExternalFiles(video, directoryService, false).Select(info => info.Path));
                if (set.Count > 0)
```